components/seventodos/seventodos.app.mjs

```diff
@@ -1,11 +1,41 @@
+// The API from 7todo is rejecting the pipedream's axios
+import axios from "axios";
+
 export default {
-    type: "app",
-    app: "seventodos",
-    propDefinitions: {},
-    methods: {
-      // this.$auth contains connected account data
-      authKeys() {
-        console.log(Object.keys(this.$auth));
-      },
-    },
-  };
\ No newline at end of file
+  type: "app",
+  app: "seventodos",
+  propDefinitions: {},
+  methods: {
+    _getBaseUrl(endpoint) {
+      return `https://7todos.com/api/v1/${endpoint}`;
+    },
+    _getHeaders() {
+      return {
+        "content-type": "application/json",
+        "key": `${this.$auth.key}`,
+        "workspaceId": `${this.$auth.workspaceId}`,
+      };
+    },
+    objRemoveUndefinedProps(obj) {
+      for (const key of Object.keys(obj)) {
+        if (obj[key] === undefined || obj[key] === null) {
+          delete obj[key];
+        }
+      }
+      return obj;
+    },
```

Comment:
Optional:
I always tend to filter instead of `delete`ing object's properties so here it is my suggestion for this function:
```suggestion
    filterEmptyValues(obj) {
        return Object.entries(obj)
          .reduce((reduction, [key, value]) => {
            if (value === undefined || value === null) {
              return reduction;
            }
            return {
              ...reduction,
              [key]: value,
            };
          }, {});
  }
```

Reply:
Thank you, just adding: the way I wrote this function, for me, seems easier to understand but I changed to the way that you suggested.